GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -634,7 +639,7 @@ private async Task ShowSelectedFileDiffAsync(bool ensureNoSwitchToFilter, int? l
             await DiffText.ViewChangesAsync(DiffFiles.SelectedItem,
                 line: line,
                 openWithDiffTool: () => firstToSelectedToolStripMenuItem.PerformClick(),
-                additionalCommandInfo: (DiffFiles.SelectedItem?.Item?.IsRangeDiff is true) && Module.GitVersion.SupportRangeDiffPath ? _revisionGrid.CurrentFilter.PathFilter : "",
+                additionalCommandInfo: (DiffFiles.SelectedItem?.Item?.IsRangeDiff is true) && Module.GitVersion.SupportRangeDiffPath ? _pathFilter() : "",
```

Comment:
For the paranoid, add `Validates.NotNull(_pathFilter);`?

Reply:
adding